config/database.yml

```diff
@@ -23,16 +23,16 @@ default: &default
 
 development:
   <<: *default
-  database: blog_development
+  database: blog
 
   # The specified database role being used to connect to postgres.
   # To create additional roles in postgres see `$ createuser --help`.
   # When left blank, postgres will use the default role. This is
   # the same name as the operating system user running Rails.
-  username: bunbee
+  username: blog
```

Comment:
- [ ]  [OPTIONAL] It's not a good idea to write your username and password in the database.yml file